scripts/build.sh

```diff
@@ -55,6 +55,16 @@ function run::build() {
 
       echo "Success!"
 
+      printf "%s" "Building helper... "
+
+      GOOS=linux \
+        go build \
+          -ldflags="-s -w" \
+          -o "helper" \
+            "${BUILDPACKDIR}/helper/run"
+
+      echo "Success!"
+
```

Comment:
@paketo-buildpacks/dotnet-core-maintainers will this get overridden by our "update github-config" workflows?

Reply:
yes, unless we add this file to the syncignore

Reply:
I think that https://github.com/paketo-buildpacks/dotnet-execute/pull/277#issuecomment-1064163463 has the solution, I will revert this change.